src/compas/geometry/_core/normals.py

```diff
@@ -59,7 +60,7 @@ def normal_polygon(polygon, unitized=True):
     for i in range(p):
         b = polygon[i]
         ob = subtract_vectors(b, o)
-        n = cross_vectors(oa, ob)
+        n = scale_vector(cross_vectors(oa, ob), 0.5)
```

Comment:
would be better to add this as part of the return statement instead of doing this multiplication over and over again

Reply:
```python
if not unitized:
    return 0.5 * nx, 0.5 * ny, 0.5 * nz
```